tools/mdalDummyExternalDriver.cpp

```diff
@@ -0,0 +1,116 @@
+/*
+ MDAL - Mesh Data Abstraction Library (MIT License)
+ Copyright (C) 2020 Vincent Cloarec (vcloarec at gmail dot com)
+*/
+
+#include <map>
+#include <fstream>
+#include <vector>
+
+#include "mdal.h"
+#include "mdal_external_driver.h"
+
+
+static std::map<int, MDAL_MeshH> sMeshes;
+static int sIdGenerator = 0;
+
+
+const char *MDAL_DRIVER_driverName()
+{
+  return "DUMMY";
+}
+const char *MDAL_DRIVER_driverLongName()
+{
+  return "Dummy dynamic driver";
+}
+const char *MDAL_DRIVER_filters()
+{
+  return "";
+}
+int MDAL_DRIVER_capabilities()
+{
+  return 0;
+}
+
+bool MDAL_DRIVER_canReadMesh( const char *uri )
+{
+  MDAL_MeshH m = MDAL_LoadMesh( uri );
+  return ( m != nullptr );
+
+  MDAL_CloseMesh( m );
```

Comment:
from driver you cannot all mdal api functions at all, you need to "impleneent" these functions. so for example for 2dm dynamic driver you would have in this function this code https://github.com/lutraconsulting/MDAL/blob/455d1aee9f901484e740a920ab646594e83183b4/mdal/frmts/mdal_2dm.cpp#L94